templates/default/footer.tpl.html

```diff
@@ -1,3 +1,9 @@
 <footer>
-    <a href="mailto:<%= contact %>"><%=contact%></a>
-</footer>
\ No newline at end of file
+        <div>
+                <%= town %>
+                <%= address %>
+            </div>
+    <a href="mailto:<%= email %>" class="email"><%= email %></a>
+    <!--Link through phone number?-->
+    <a><%= phone %></a>
```

Comment:
It's nice that you turn the phone number into a link. But please do it properly.

Check: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a